src/views/Restoration/RestorationReviewConfirm.vue

```diff
@@ -60,6 +60,18 @@
         />
       </v-card>
     </section>
+
+    <!-- Staff Payment -->
+    <section id="staff-payment-section" class="mt-10" v-if="isRoleStaff">
+      <header>
+        <h2>Staff Payment</h2>
+        <p class="mt-4"></p>
+      </header>
+
+      <v-card flat class="mt-6">
+        <StaffPayment class="py-8 px-6" />
+      </v-card>
+    </section>
```

Comment:
I added this so I could test that the staff payment options work. They do!

![image](https://user-images.githubusercontent.com/10002889/213031336-816bd7a2-1580-47f2-8083-a40cd06bd7cb.png)

![image](https://user-images.githubusercontent.com/10002889/213031380-ada1898c-2c38-4881-af45-314971bc2ba3.png)